services/QuillLMS/client/app/bundles/Staff/components/evidence/healthDashboards/activityHealthDashboard.tsx

```diff
@@ -12,6 +12,7 @@ import { addCommasToThousands, getLinkToActivity, secondsToHumanReadableTime } f
 import { fetchAggregatedActivityHealths } from '../../../utils/evidence/ruleFeedbackHistoryAPIs';
 
 const ALL_FLAGS = "All Flags"
+const PRODUCTION_FLAG = "production"
```

Comment:
Very minor, but you can actually export PRODUCTION_FLAG from the shared bundle, which is probably worth doing here (just add `, PRODUCTION_FLAG` after `filterNumbers` in line 10).